src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -19,9 +19,9 @@
     <!-- the following package(s) are from https://github.com/dotnet/fxdac -->
     <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />
     <!-- the following package(s) are from https://github.com/dotnet/corefx -->
-    <PackageReference Include="System.IO.Packaging" Version="8.0.0-preview.4.23259.5" />
-    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="8.0.0-preview.4.23259.5" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="8.0.0-preview.4.23259.5" />
+    <PackageReference Include="System.IO.Packaging" Version="8.0.0-preview.7.23375.6" />
+    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="8.0.0-preview.7.23375.6" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="8.0.0-preview.7.23375.6" />
     <!-- the following package(s) are from https://github.com/dotnet/wcf -->
```

Comment:
Can you please add a new comment here calling out their version is pinned and link to the PowerShell issue?